docs/install/manual-install.md

```diff
@@ -132,32 +131,12 @@ and compiles any translation `.mo` files (using `commonlib/bin/gettext-makemo`).
 
 #### macOS troubleshooting
 
-Some Perl modules may fail to install without certain packages and paths in place.
-
-First make sure you have installed any [prerequisite packages](#pre-req-pkg-mac).
-
-For any **SSL-related modules**, you may need to create symlinks to folders Perl modules will recognise:
-
-{% highlight bash %}
-$ sudo mkdir /usr/local/lib
-$ sudo ln -s $HOMEBREW_PREFIX/opt/openssl@1.1/lib/libssl.1.1.dylib /usr/local/lib/libssl.dylib
-$ sudo ln -s $HOMEBREW_PREFIX/opt/openssl@1.1/lib/libcrypto.1.1.dylib /usr/local/lib/libcrypto.dylib
-{% endhighlight %}
-
-You may also need to set one or more of the following env variables:
-
-{% highlight bash %}
-$ export OPENSSL_PREFIX=$HOMEBREW_PREFIX/opt/openssl@1.1
-$ export PKG_CONFIG_PATH="$HOMEBREW_PREFIX/opt/openssl@1.1/lib/pkgconfig"
```

Comment:
Are PKG_CONFIG_PATH, LDFLAGS, and CPPFLAGS not needed at all now?

Reply:
No, on a fresh install on a new Mac, OPENSSL_PREFIX was all that was needed to locate all the SSL related gubbins.